components/UI/navbar.tsx

```diff
@@ -13,7 +13,7 @@ import { useDisplayName } from "../../hooks/displayName.tsx";
 
 const Navbar: FunctionComponent = () => {
   const [nav, setNav] = useState<boolean>(false);
-  const [hasWallet, setHasWallet] = useState<boolean>(true);
+  const [hasWallet, setHasWallet] = useState<boolean>(false);
```

Comment:
Is it necessary ? As I tested, the auto-connect works well 🔥

But when you are new to the app, you have to click on connect on the top right button in order to open this. It's a better UX to directly open it only if the user never connected imo. 

Wdyt ?

Reply:
Me, I prefer when it doesn't popup even when I'm new, but that's my personal point of view :)

If it is set to true at first and the user has already connected once then the popup is shown for less than a second before it disappears. What I could do is to keep it to `false` whenever a user connects, and check local storage: 
- if there is a value for `lastUsedConnector` then `hasWallet` stays to `false`, 
- if there is no value, set `hasWallet` to `true` and show the popup wallet.

Reply:
Or maybe we could directly connect the user to the first wallet on the list if he is not connected? This means it'll directly open his Argent/Braavos app and ask him to approve the connection.